addons/mail/models/mail_message.py

```diff
@@ -1090,3 +1093,18 @@ def _invalidate_documents(self, model=None, res_id=None):
 
     def _get_search_domain_share(self):
         return ['&', '&', ('is_internal', '=', False), ('subtype_id', '!=', False), ('subtype_id.internal', '=', False)]
+
+    def _is_empty_message(self):
+        self.ensure_one()
+        is_body_empty = not self.body or is_html_empty(self.body)
+        is_subtype_description_empty = not self.subtype_id or not self.subtype_id.description or is_html_empty(html_sanitize(self.subtype_id.description, strip_style=True, strip_classes=True))
```

Comment:
I don't think we have to use html sanitize. Either it is empty from our definition (void or pseudo-editor-void), either we should consider it as "not empty". Anyway I would remove completely the description usage as explained before.